cmd/ui/src/views/Explore/GraphView.tsx

```
@@ -52,6 +53,11 @@ import ExploreSearch from './ExploreSearch/ExploreSearch';
 import GraphItemInformationPanel from './GraphItemInformationPanel';
 import { transformIconDictionary } from './svgIcons';
 
+const makeMap = (items: any[]) =>
+    items.reduce((acc, col) => {
+        return { ...acc, [col?.accessorKey || col?.id || 'accessor_key_unavailable']: true };
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)